packages/babylon/src/types.js

```diff
@@ -22,15 +22,15 @@ export type Comment = {
 };
 
 export interface NodeBase {
-  start: number,
-  end: number,
-  loc: SourceLocation,
-  range: [number, number],
-  leadingComments?: ?Array<Comment>,
-  trailingComments?: ?Array<Comment>,
-  innerComments?: ?Array<Comment>,
-
-  extra: { [key: string]: any },
+  start: number;
+  end: number;
+  loc: SourceLocation;
+  range: [number, number];
+  leadingComments?: ?Array<Comment>;
+  trailingComments?: ?Array<Comment>;
+  innerComments?: ?Array<Comment>;
+
+  extra: { [key: string]: any };
```

Comment:
I don't know why here there are the semicolons: `make fix` adds them but my editor (which loads the correct eslint config) removes them 🤔

Reply:
Prettier recently forces semi's on interfaces:
https://github.com/prettier/prettier/issues/2593